src/main/java/com/vg/hm/samples/usermanagementsvc/service/impl/VehicleServiceImpl.java

```diff
@@ -0,0 +1,51 @@
+package com.vg.hm.samples.usermanagementsvc.service.impl;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.netflix.hystrix.contrib.javanica.annotation.HystrixCommand;
+import com.vg.hm.samples.usermanagementsvc.service.VehicleAPIService;
+import com.vg.hm.samples.usermanagementsvc.service.VehicleService;
+import com.vg.hm.samples.usermanagementsvc.service.model.Make;
+import com.vg.hm.samples.usermanagementsvc.service.model.MakesJson;
+import lombok.extern.log4j.Log4j2;
+import org.springframework.stereotype.Service;
+import retrofit2.Call;
+import retrofit2.Response;
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+@Log4j2
+@Service
+public class VehicleServiceImpl implements VehicleService {
+    public static final String BASE_URL = "https://vpic.nhtsa.dot.gov/api/vehicles/";
+    Gson gson = new GsonBuilder().setDateFormat("yyy-MM-dd'T'HH:mm:ssz").create();
```

Comment:
This should be moved to a config class with Spring, and declare a bean for this, then you can inject this dependency here.